package.json

```diff
@@ -15,12 +15,15 @@
   },
   "dependencies": {
     "argon2": "^0.28.2",
+    "cors": "^2.8.5",
```

Comment:
This isn't necessary, I can help you fix your webpack config so that everything works on localhost:3000 normally.

```js
// add this to your webpack.config.js proxy settings and restart your dev script
'/socket.io': `http://localhost:${process.env.PORT}`
```

it goes underneath the `'/api'` setting in the `proxy` property.